docs/src/pages/docs/functions/combinators.md

```diff
@@ -155,11 +312,50 @@ as an exercise to reason about why this is so powerful and important.
 substitution :: (a -> b -> c) -> (a -> b) -> a -> c
 ```
 
-While it a complicated little bugger, it can come in very handy from time to
-time. In it's first two arguments it takes functions. The first must be binary
-and the second unary. That will return you a function that is ready to take some
-value. Once supplied the fun starts, it will pass the `a` to the first argument
-of both functions, and the result of the second function to the second parameter
-of the first function. Finally after all that juggling, it will return the
-result of that first function. When used with partial application on that first
-parameter, a whole new world of combinatory madness is presented!
+While it may seem like a complicated little bugger, `substitution` can come in
+very handy from time to time. `substitution` is used when you have a `binary`
```

Comment:
lines with code cannot end or start on a line, the docs will not properly catch the linebreak and enter a space.